source/SymWorld.h

```diff
@@ -240,6 +280,44 @@ class SymWorld : public emp::World<Organism>{
     file.AddHistBin(node, 19, "Hist_0.9", "Count for histogram bin 0.9 to 1.0");
 
 
+    file.PrintHeaderKeys();
+
+    return file;
+  }
+  emp::DataFile & SetupSggHostIntValFile(const std::string & filename) {
```

Comment:
I don't think you need this function, since there isn't any new host-specific data associated with the SGG stuff